components/forms/welcome-card-form.tsx

```diff
@@ -46,6 +46,24 @@ interface WelcomeCardFormProps {
   userId: string
 }
 
+const placeholders = [
+  {
+    frontTitle: "何をするサークル？",
+    backTitle: "楽しくプログラミングするサークル!",
+    backDescription: "みんなで集まってプログラムをしています!\n\n★うんたら\n★かんたら\n\nみたいな例文ほしいです!\n・カード1つに収まる文章で入力量決められるといいな\n画像ファイルもアルバム詳細みたい"
```

Comment:
```suggestion
    backDescription: "みんなで集まってプログラムをしています!\n\n★勉強するだけでなく、ハッカソンや個人開発などで実践を積み重ね切磋琢磨しています。\n\n★初心者でも一緒に成長していける環境なのでぜひ一緒に取り組みましょう。\n\nみたいな例文ほしいです!\n・カード1つに収まる文章で入力量決められるといいな\n画像ファイルもアルバム詳細みたい"
```

```
★うんたら
★かんたら
```

この箇所だけ少し適当すぎる気がするので、

```
★勉強するだけでなく、ハッカソンや個人開発などで実践を積み重ね切磋琢磨しています。
★初心者でも一緒に成長していける環境なのでぜひ一緒に取り組みましょう。
```

みたいな文言にしてみても良いかも